src/algebra/homology/local_cohomology.lean

```diff
@@ -0,0 +1,137 @@
+/-
+Copyright (c) 2023 Emily Witt. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Emily Witt, Scott Morrison
+-/
+import ring_theory.ideal.basic
+import algebra.category.Module.colimits
+import algebra.category.Module.projective
+import category_theory.abelian.ext
+import category_theory.limits.final
+
+/-!
+# Local cohomology.
+
+We just give one definition, deferring for later the other equivalent definitions,
+and basic properties.
+-/
+
+open opposite
+open category_theory
```

Comment:
```suggestion
open opposite category_theory
```

Reply:
Thanks for the suggestion; I would prefer to leave as-is